src/main/resources/application.conf

```diff
@@ -37,26 +37,30 @@ backend {
   backend = "local" // Either "JES" or "local", case insensitive
 
   // If backend is JES, must supply a 'jes' stanza:
-  //      jes
-  //          applicationName
-  //          project (i.e. google project name)
-  //          baseExecutionBucket (GCS path where things will be written to)
+  jes {
+    applicationName = ""
+    project = ""
+    baseExecutionBucket = ""
+  }
 }
 
 // If authenticating with Google (e.g. if backend is 'JES') you must supply a top level stanza 'google':
+google {
+  authScheme = "user"
 
-//      google
-//           authScheme  - Either "service" for service account or "user" for user-based, case insensitive
-                // If "user" you must supply:
-                //      userAuth
-                //           user
-                //           secretsFile
-                //           dataStoreDir (dir to cache oauth credentials)
-                // You will be asked to do the oauth browser dance
+  // If authScheme is "user"
+  userAuth {
+    user = ""
+    secretsFile = ""
+    dataStoreDir = ""
+  }
 
-                // If "service" you must supply:
-                //        p12File
-                //        serviceAccountId
+  // If authScheme is "service"
+  serviceAuth {
+    p12File = ""
+    serviceAccountId = ""
+  }
+}
 
 database {
   config = main.hsqldb
```

Comment:
My main objection was to having "live" conf stuff in there for the JES stuff, that seems like a bug waiting to happen. I already think my comment pseudo-doc is teh suck so if you think some other format would have been easier for you to figure out what's going on you'd know better than me since you had to just navigate it :)